### PR TITLE
mu: update to 1.2

### PR DESCRIPTION
**Pull request by emacsomancer**

### CODE REVIEW

File: srcpkgs/mu/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+revision=1
 build_style=gnu-configure
 hostmakedepends="automake libtool pkg-config emacs texinfo"
-makedepends="xapian-core-devel gmime-devel libuuid-devel"
-short_desc="A tool for dealing with e-mail messages stored in the Maildir-format"
+makedepends="xapian-core-devel gmime3-devel libuuid-devel"
+short_desc="Emacs-based tool for dealing with e-mail stored in the Maildir-format"
 maintainer="Benjamin Slade <slade@jnanam.net>"
-homepage="http://www.djcbsoftware.nl/code/mu/"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/mu/template]: use spdx identifier for licens

**Comment by maxice8** [comment]
commit should have pkgname= before the :

**Comment by maxice8** [comment]
oops pressed wrong button

**Comment by emacsomancer** [comment]
> commit should have pkgname= before the :

mu4e is linked to mu, so that's why I did 'mu/mu4e'. Should it just be 'mu'? (Or just 'mu4e'?)

**Comment by maxice8** [comment]
name of the main package, not any subpackages.

**Pull request merged**